clang/tools/dpct/DpctOptRules/cmake_script_migration_rule.yaml

```diff
@@ -748,3 +748,55 @@
   CmakeSyntax: cuda_add_library
   In: cuda_add_library(${arg})
   Out: add_library(${arg})
+
+- Rule: rule_target_compile_features-cuda_std_version
+  Kind: CMakeRule
+  Priority: Fallback
+  CmakeSyntax: target_compile_features-cuda_std-version
+  In: target_compile_features(${arg})
+  Out: target_compile_features(${arg})
+  Subrules:
+    arg:
+      In: cuda_std_${version}
+      Out: cxx_std_17
+      MatchMode: Partial
+      RuleId: "replace_cuda_std_version_with_cxx_std_version"
+
+- Rule: rule_set_property-cuda_standard
+  Kind: CMakeRule
+  Priority: Fallback
+  CmakeSyntax: set_property-cuda_standard
+  In: set_property(${target} PROPERTY CUDA_STANDARD ${version})
+  Out: set_property(${target} PROPERTY CXX_STANDARD ${version})
+  Subrules:
+    version:
+      In: ${version}
+      Out: 17
+      MatchMode: Partial
+      RuleId: "replace_set_property_cuda_stardard_with_cxx_standard"
+
+- Rule: rule_set_target_properties-cuda_standard
+  Kind: CMakeRule
+  Priority: Fallback
+  CmakeSyntax: set_target_properties-cuda_standard
+  In: set_target_properties(${targets} PROPERTIES ${properties})
+  Out: set_target_properties(${targets} PROPERTIES ${properties})
+  Subrules:
+    properties:
+      In: CUDA_STANDARD ${version} ${other_properties}
+      Out: CXX_STANDARD 17 ${other_properties}
+      MatchMode: Full
+
+- Rule: rule_set_target_properties-cuda_standard-only
+  Kind: CMakeRule
+  Priority: Fallback
+  CmakeSyntax: set_target_properties-cuda_standard-only
+  In: set_target_properties(${targets} PROPERTIES${properties}CUDA_STANDARD ${version})
+  Out: set_target_properties(${targets} PROPERTIES${properties}CXX_STANDARD 17)
```

Comment:
the lines has function overlap with line140~177, pls simple these migration rules to remove duplicates.

Reply:
Removed.